deploy/vagrant/modules/javascript/templates/install_phantomjs.erb

```diff
@@ -1,13 +0,0 @@
-#!/bin/bash
-##### Install phantomjs
-
-set -e
-set -x
-
-PHANTOMJS_VERSION_NAME="phantomjs-1.9.7-linux-x86_64"
-
-cd /tmp
-/usr/bin/wget https://bitbucket.org/ariya/phantomjs/downloads/${PHANTOMJS_VERSION_NAME}.tar.bz2 -O /tmp/${PHANTOMJS_VERSION_NAME}.tar.bz2
-sleep 10
-/bin/tar xjf ${PHANTOMJS_VERSION_NAME}.tar.bz2
-cp ${PHANTOMJS_VERSION_NAME}/bin/phantomjs /usr/local/bin/phantomjs
```

Comment:
This was no longer used, so i deleted it to avoid confusion around how we're installing phantomjs.